test/test_css_parser_media_types.rb

```diff
@@ -8,6 +8,25 @@ def setup
     @cp = Parser.new
   end
 
+  def test_that_duplicate_media_queries_do_not_duplicate_rules
+    query_to_duplicate = ["only screen and (max-width: 480px)"]*100
```

Comment:
Is it necessary to do this 100 times, or would 2 do the trick?

Reply:
2 would do the trick. I think it makes no difference though. I will follow up in a bit